src/app/contracts/publish/publish-contract/page.mdx

```diff
@@ -0,0 +1,85 @@
+
+
+# Publish
+
+Use the [CLI](/cli) to publish your contracts directly to **Any EVM compatible chain** using the [dashboard](/dashboard):
+
+```bash
+npx thirdweb publish
+```
+
+This command will:
+
+- Compile your smart contract.
+- Upload the contract source code (ABI) to IPFS.
+- Open the publish flow in the dashboard.
+
+
+## How to Publish Your Contract
+
+In your browser, follow the steps below to publish your contract:
+
+### Configure Contract Metadata
+
+From the dashboard, provide the metadata for your smart contract including:
+
+- **Name**
+- **Image**
+- **Description**
+- **README** - Markdown is supported. What your contract does, common use cases & examples are great things to include here.
+
+![Enter Contract Metadata](/assets/publish/publish-your-contract.png)
```

Comment:
broken images on this page

Reply:
they are old screenshots- going to delete for now and re-add next week